app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+
+  <h2>Log in</h2>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
```

Comment:
- [ ] Great job working on the sign in page but please as shown in the guidelines your design should be the same as the given design so kindly make the layout the same including the colors as mentioned in the reqiorements ⭐🚀🔥

![image](https://github.com/Timbar09/Budget-App/assets/58553711/2e2e9455-9996-40db-a41b-c5d8751e6c04)


## please change the color as well 🔥💪🏻💪🏻
- Your
![image](https://github.com/Timbar09/Budget-App/assets/58553711/29ab37fd-574e-471e-a95a-0e8b6459dc84)

- Given
![image](https://github.com/Timbar09/Budget-App/assets/58553711/694793ff-c2e7-4065-8282-347a14cfe682)